src/common/zscore/cgmzscore/src/calculate.py

```diff
@@ -0,0 +1,67 @@
+'''https://www.who.int/childgrowth/standards/Chap_7.pdf'''
+from decimal import Decimal as D
+
+
+class Zscore:
+    '''Class to do calculation of z score'''
```

Comment:
We use """ instead of ''' in most places.

```suggestion
    """Class to do calculation of z score"""
```

Also replace this in other docstrings to be consistent